kernel-modules/support-packages/01-collector-to-rox-version-map.py

```diff
@@ -25,6 +27,39 @@ def parse_released_versions(f):
 
     return result
 
```

Comment:
[ultra nit] Use two blank lines between functions. Check if your script is pep8 compliant.

Reply:
I'll make sure the modified scripts are pep8 complaint, though to be clear they were already non-complaint before my changes. I believe #654 is adding pep8 enforcement, so that should help us in the near future.